src/modules/product/product.route.js

```
@@ -1,20 +1,68 @@
 import { Router } from "express";
-import { userMiddleware } from "../user/user.middleware.js";
+import { userMiddleware, adminMiddleware } from "../user/user.middleware.js";
 import ProductControllers from "./product.controller.js";
 import { IPRateLimiter } from "../../common/config/rate-limit.js";
 import { cacheMiddleware } from "../../common/middlewares/cache.middleware.js";
+// import { authMiddleware } from "../../common/middlewares/auth.middleware.js";
+import multer from 'multer';
+import path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)